src/Microsoft.PowerApps.TestEngine/PowerFx/PowerFxEngine.cs

```diff
@@ -28,6 +28,12 @@ public class PowerFxEngine : IPowerFxEngine
 
         private RecalcEngine Engine { get; set; }
         private ILogger Logger { get { return _singleTestInstanceState.GetLogger(); } }
+        private static ParserOptions TestEngineParserConfig(CultureInfo culture)
+        {
+            // Currently support for decimal is in progress for PowerApps
+            // Power Fx by default treats number as decimal. Hence setting NumberIsFloat config to true in our case
+            return new ParserOptions() { AllowsSideEffects = true, Culture = culture, NumberIsFloat = true };
```

Comment:
nit: consider defining these parameters as class members, that way it helps with reuse and enhances readability.

Reply:
nit: move this function towards the end of the class. lol for a minute there, i thought it is a static property on the class. some separation between fields and methods would be nice. also name it something like GetPowerFxParsingOptions.